### PR TITLE
Adds nardiello Web Page for the Course web site

### PR DESCRIPTION
**Pull request by nardielloam7451**

### CODE REVIEW

File: students/index.html [hunk 1]
```diff
@@ -10,6 +10,7 @@
     <h1>Methods 2021 - Students</h1>
     <ul>
       <li><a href="./example.html">Example Student</a></li>
+	    <li><a href="./nardielloam.html">Anthony Nardiello</a></li>
```
Comment (sleddog) [inline students/index.html]: nit - consistent whitespace

File: students/nardielloam.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+</head>
+<body>
+    <h1>Example Student</h1>
+    <p>Hi!! My name is Anthony Nardiello, a 4th year Undergraduate Student at Montana State University, majoring with a Bachelor's of Science in Computer Science. My interests within Computer Science include
+	Machine Learning, Artificial Intelligence, Systems Architecture, and Software Engineering!! Some of my hobbies outside of my studies include reading, writing, cooking, tea, amateur photography, and travel (when we can more safely
+	Travel). Check below to see my Linkedin and Github for professional development, or follow me on Twitter for more personal insights into life and the world around us! </p>
+    <ul>
+      <li><a href="https://www.linkedin.com/in/anthony-nardiello-47a965139/">Check out my Linkedin</a></li>
+      <li><a href="https://github.com/nardielloam7451">Check out my Github</a></li>
+	  <li><a href="https://twitter.com/Futurist7451">Check here for my Twitter Ramblings</a></li>
```
Comment (sleddog) [inline students/nardielloam.html]: same line up the <li> tags

Comment (nardielloam7451) [reply students/nardielloam.html]: Latest commit fixes this issue by lining them up!

**Review by sleddog — APPROVED** [review]
Whitespace still looks a little off.  I'll add this to a discussion topic next class, but works for now.

**Review by sleddog — APPROVED** [review]

**Pull request merged**